### ISSUE TITLE
[JS] Bumping automatically package.json file with Semver

### ISSUE THREAD

**Issue by iarroyo5** [dependabot/dependabot-core#1191]
**Context**:
- Using _dependabot_ against a private _Gemfury_ `npm` repository
- Using _semver_ as semantic versioner for bumping our private libraries
- Working on private library called `A`
- Libraries to be bumped by dependabot `XYZ`

**Issue**:

Following the documentation from [here](https://dependabot.com/docs/config-file/#automerged_updates) I could see there are existing options for 
- `automerged_updates` and 
- `version_requirement_updates`

but both options are for updating the versions of the libraries used inside the `dependencies` or `devDependencies` (`XYZ` in my context), my issue is even if these two features are nice, we are using `semver` for versioning our libraries, so every time one of this `XYZ` dependencies are detected and bumped by _dependabot_, in order to be able to release the "new version of the library `A`" we need to bump the version inside the `pacakge.json`, let's say we are doing `npm version patch` for this kind of updates...

Is there any kind of configuration which I may miss inside `.dependabot config` file to execute this automatically and bump the `package.json` and the `package-lock.json` files inside the library `A`?

**Comment by greysteil** [comment]
Thanks for such a detailed writeup.

Unfortunately we don't support that, and don't have any plans to (just because we think of it as slightly out of scope). Do you have access to the GitHub Actions beta? This feels like a perfect use case for that, and you could perhaps re-use some of the logic from the `PullRequestCreator` and `FileFetcher` in [Dependabot Core](https://github.com/dependabot/dependabot-core) (if you want to use Ruby).

Sorry not to be of more help, but I wouldn't want you waiting on us for this when it's not something we're planning!

**Comment by iarroyo5** [comment]
Thanks to you @greysteil for the quick response and the honestly. 

I'll have a look the `PullRequestCreator` and `FileFetcher` 👍
